listenbrainz/webserver/templates/user/scraper.js

```diff
@@ -66,6 +87,14 @@ Scrobble.prototype.trackName = function () {
     }
 };
 
+Scrobble.prototype.releaseName = function() {
+    if ('album' in this.rootScrobbleElement && '#text' in this.rootScrobbleElement['album']) {
+        return this.rootScrobbleElement['album']['#text'];
+    } else {
+        return "";
```

Comment:
it looks like we have as much of a problem with quote styles in js as we do in python. We should be a bit more careful in the future with consistency.